galleries/examples/misc/patheffect_demo.py

```diff
@@ -29,7 +29,8 @@
 ax2.imshow(arr)
 cntr = ax2.contour(arr, colors="k")
 
-cntr.set(path_effects=[patheffects.withStroke(linewidth=3, foreground="w")])
+for collection in cntr.collections:
```

Comment:
The use of the `collections` attribute actually is deprecated and will no longer be available in the upcoming release.  Since the `ContourSet` is now itself a collection, you can simply use `cntr.set_path_effects(…)`